docs/list.md

```diff
@@ -330,6 +330,21 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### N
 
+#### **NativeScript**
+
+- **Swag**: Exclusive NativeScript Swags
+- **Requirements**:
+    - Create a StackBlitz of anything fun or neat about [NativeScript](https://preview.nativescript.org){:target="\_blank"}.
+    - Contribute a PR for an issue in a participating NativeScript repository.
+    - Share a gist of neat NativeScript code you find useful.
+    - Create a NativeScript plugin.
+    - Tweet about it and mention @NativeScript.
+    - DM us your address to get free swag.
```

Comment:
Please don't DM _me_ your address! 😆 Please make this clear that NativeScript is looking for a DM.